### DISCUSSION TITLE
what version is android studio

### DISCUSSION THREAD

**Discussion by CodeBhaukaal** [capcom6/android-sms-gateway#96] [General]
what version is android studio

**Comment by CodeBhaukaal** [comment]
android studio Operation is not supported for read-only collection
problem
![Uploading sdasdsa.JPG…]()

**Comment by CodeBhaukaal** [comment]
![sdasdsa](https://github.com/user-attachments/assets/542ecc1a-5d25-478d-a4cd-f7f57559529f)

**Reply by capcom6** [reply]
Unfortunately, the provided screenshot does not offer much information about the issue. One possible cause could be the enabling of experimental Gradle features. For more details, see https://stackoverflow.com/questions/77583994/android-studio-operation-is-not-supported-for-read-only-collection.

If the solution from Stack Overflow doesn't resolve the problem, please try clicking on each line in the left panel of the Build Output tab to obtain more detailed information, and provide that to me.

For clarity, I use different versions of Android Studio on different devices. Today, I have tested on Android Studio Hedgehog | 2023.1.1 Patch 2.
